### PR TITLE
[ready] Feature/tasks filter

### LINKED ISSUE #213: Allow users to filter Tasks to a single admin area
This seems like a reasonable request; if a provincial or district official wants to work on fixes only for their admin area, they should be able to filter out all Tasks that don't affect them.

### PR DESCRIPTION
**Pull request by geohacker**
Fixes #213 

image

This completes the series of PRs for #213:
* https://github.com/orma/openroads-vn-api/pull/101
* https://github.com/orma/openroads-vn-tiler/pull/39

@ricardomestre - could you help with figuring out a good way to position the Select input? It could be above the panel but I wonder if you have ideas. Thank you!

cc @kamicut

### CODE REVIEW

**Comment by ricardomestre** [comment]
@geohacker The position seems good to me. I just added a bit of margin to the element. This is good to go!

**Comment by geohacker** [comment]
@ricardomestre thank you! this looks great 👌 

image

@kamicut can you give this a quick review?

**Comment by geohacker** [comment]
@kamicut thank you for helping get this across the line!

File: app/assets/scripts/redux/store.js [hunk 1]
```diff
@@ -3,13 +3,14 @@ import thunkMiddleware from 'redux-thunk';
...
+import logger from 'redux-logger';
 
 
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 
 export default createStore(
   reducer,
   composeEnhancers(
-    applyMiddleware(syncHistory(hashHistory), thunkMiddleware)
+    applyMiddleware(logger, syncHistory(hashHistory), thunkMiddleware)
```
Comment (kamicut) [inline app/assets/scripts/redux/store.js]: I feel like this should be a configuration variable at build time. @danielfdsilva can help here.

File: app/index.html [hunk 2]
```diff
@@ -43,6 +43,8 @@
 
     <link href='https://api.mapbox.com/mapbox.js/v2.3.0/mapbox.css' rel='stylesheet' />
     <link href='https://api.mapbox.com/mapbox-gl-js/v0.39.1/mapbox-gl.css' rel='stylesheet' />
+    <link rel="stylesheet" href="https://unpkg.com/react-select/dist/react-select.css">
```
Comment (kamicut) [inline app/index.html]: Is the CSS being packaged using the vendor CSS scripts, or inserted globally? I feel there is a redundancy here unless I'm not understanding how you imported the CSS.

Comment (geohacker) [reply app/index.html]: It's not packed. I just went with loading how it works for the mapbox vendor.

File: app/assets/scripts/redux/store.js (hunk 1 above)
Comment (geohacker) [reply app/assets/scripts/redux/store.js]: Oh right, I forgot to take that out. I think there's a `process.env.DS_ENV === 'staging'` check common for picking up staging configs. I'll make the logger part of staging.

File: app/index.html (hunk 2 above)
Comment (kamicut) [reply app/index.html]: Ok, then remove the react-select scss imports I guess?

Comment (geohacker) [reply app/index.html]: I can't figure out why it doesn't seem to get built for me. I've asked @ricardomestre for help. But I might merge this and patch that in later.

**Pull request merged**